packages/permission-log-controller/tests/PermissionLogController.test.ts

```diff
@@ -366,97 +368,101 @@ describe('PermissionLogController', () => {
           RPC_REQUESTS.custom(SUBJECTS.b.origin, 'eth_getBlockNumber'),
-          res,
-          mockNext,
-          noop,
-        );
-        logMiddleware(
           RPC_REQUESTS.custom(SUBJECTS.b.origin, 'net_version'),
-          res,
-          mockNext,
-          noop,
-        );
+        ];
+
+        ignoredMethods.forEach((req) => {
+          logMiddleware(req, res, mockNext, noop);
+        });
 
         expect(controller.state.permissionActivityLog).toHaveLength(0);
       });
 
-      it('enforces log limit', () => {
-        const req = RPC_REQUESTS.test_method(SUBJECTS.a.origin);
-        const res = {
-          ...PendingJsonRpcResponseStruct.TYPE,
-          result: ['bar'],
-        };
+      describe('log limit enforcement', () => {
+        it('fills up the log to its limit without exceeding', () => {
+          const controller = initController();
+          const logMiddleware = initMiddleware(controller);
+          const req = RPC_REQUESTS.test_method(SUBJECTS.a.origin);
+          const res = { ...PendingJsonRpcResponseStruct.TYPE, result: ['bar'] };
 
-        // max out log
-        let lastId;
-        for (let i = 0; i < LOG_LIMIT; i++) {
-          lastId = nanoid();
-          logMiddleware({ ...req, id: lastId }, res, mockNext, noop);
-        }
-
-        // check last entry valid
-        expect(controller.state.permissionActivityLog).toHaveLength(LOG_LIMIT);
-
-        validateActivityEntry(
-          controller.state.permissionActivityLog[LOG_LIMIT - 1],
-          { ...req, id: lastId ?? null },
-          res,
-          LOG_METHOD_TYPES.restricted,
-          true,
-        );
-
-        // store the id of the current second entry
-        const nextFirstId = controller.state.permissionActivityLog[1].id;
-        // add one more entry to log, putting it over the limit
-        lastId = nanoid();
+          for (let i = 0; i < LOG_LIMIT; i++) {
+            logMiddleware({ ...req, id: nanoid() }, res, mockNext, noop);
+          }
 
-        logMiddleware({ ...req, id: lastId }, res, mockNext, noop);
-
-        // check log length
-        expect(controller.state.permissionActivityLog).toHaveLength(LOG_LIMIT);
+          expect(controller.state.permissionActivityLog).toHaveLength(
+            LOG_LIMIT,
+          );
+        });
 
-        // check first and last entries
-        validateActivityEntry(
-          controller.state.permissionActivityLog[0],
-          { ...req, id: nextFirstId },
-          res,
-          LOG_METHOD_TYPES.restricted,
-          true,
-        );
+        it('removes the oldest log entry when a new one is added after reaching the limit', () => {
+          const controller = initController();
+          const logMiddleware = initMiddleware(controller);
+          const req = RPC_REQUESTS.test_method(SUBJECTS.a.origin);
+          const res = { ...PendingJsonRpcResponseStruct.TYPE, result: ['bar'] };
+
+          // Initially fill the log to its limit
```

Comment:
Nit: I feel like if we make the code clear enough, the variable names readable enough, the test is short enough, it's easy enough to follow along without needing comments to describe each test. What are your thoughts on removing them?